test/integration/fixtures/regression/issue-2286.js

```diff
@@ -0,0 +1,5 @@
+describe('suite', function () {
+  beforeEach(function () { this.skip() })
+  after(function () { console.log('after in suite') })
+  it('test', function () {})
+})
```

Comment:
(to the first reply:) Yeah, it's pretty clear from the filename what issue this is testing, but it's not so obvious to me from looking just at this file that this is meant to be run by another test in another file that tests the output of this one (unless that's what all the stuff in this fixtures folder is -- I haven't looked at the files that were already there), so it looked at first as though this weren't actually testing it programmatically but only printing something to look at manually.